src/pages/HomePage.vue

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
Why do we need separate Home page and simulator page ?

Reply:
No, we don't need it. 
All the work will be done on the simulator page.

Reply:
Removed Home component.